Sources/MobileCoinClient.swift

```diff
@@ -80,18 +80,34 @@ public final class MobileCoinClient {
         accountLock.readSync { $0.cachedTxOutTokenIds }
     }
 
-    @available(*, deprecated, message:
-        """
-        Deprecated in favor of `accountActivity(for:TokenId)` which accepts a TokenId.
-        `accountActivity` will assume the default TokenId == .MOB // UInt64(0)
+    public func recoverTransactions<Contact: PublicAddressProvider>(
+        contacts: Set<Contact>
+    ) -> [HistoricalTransaction] where Contact: Hashable {
+        recoverTransactions(allAccountActivity().txOuts, contacts: contacts)
+    }
 
-        Get a set of all tokenIds that are in TxOuts owned by this account with:
+    public func recoverContactTransactions<Contact: PublicAddressProvider>(
+        contact: Contact
+    ) -> [HistoricalTransaction] where Contact: Hashable {
```

Comment:
what about using 'for' and dropping the 'Contact' in the name, like this (same for other functions) so it reads well, like "recover transactions for contact":
```suggestion
    public func recoverTransactions<Contact: PublicdAdressProvider>(for contact: Contact
    ) -> [HistoricalTransaction] where Contact: Hashable {
```

Reply:
This type of naming was common in objective-C but I think it was dropped in swift4 or 5. So generally the parameter name should be semantic even for the first parameter. 

So I think its best to keep the current naming